cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -36,11 +36,11 @@ var (
 )
 
 func init() {
-	utilruntime.Must(componentconfig.AddToScheme(configScheme))
-	utilruntime.Must(componentconfigv1alpha1.AddToScheme(configScheme))
+	utilruntime.Must(kubeschedulerconfig.AddToScheme(configScheme))
+	utilruntime.Must(kubeschedulerconfigv1alpha1.AddToScheme(configScheme))
 }
 
-func loadConfigFromFile(file string) (*componentconfig.KubeSchedulerConfiguration, error) {
+func loadConfigFromFile(file string) (*kubeschedulerconfig.KubeSchedulerConfiguration, error) {
```

Comment:
Add a TODO that this function should error if the API group is detected to be `componentconfig/v1alpha1`. That is not supported anymore.

Reply:
not just a TODO, the existing test in TestSchedulerOptions would error and we can verify that it does

Reply:
Yeah, using test is preferable.